waspc/data/Generator/templates/sdk/server/api/index.ts

```diff
@@ -16,6 +14,8 @@ import {
   {=/ shouldImportAuthenticatedApi =}
 } from '../_types'
 
+// PUBLIC API
```

Comment:
This seemed useful -> otherwise I have to keep cross-referencing it with docs. It might not be long term solution, because that we need to sync stuff with docs, but maybe it makes sense, not sure yet. We can remove it, not biggy.